specification/cognitiveservices/data-plane/Personalizer/preview/v1.1-preview.2/Personalizer.json

```diff
@@ -0,0 +1,1860 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Personalizer Base",
```

Comment:
I believe it should be "Personalizer Client" otherwise it will be a breaking change to SDKs.

Reply:
This was suggested during the sdk review the last time we updated sdk 2 months ago.